x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/hooks.tsx

```diff
@@ -166,14 +168,21 @@ export type ActionsLogPopupFilters = Extract<
 const getTypesFilterInitialState = (
   isSentinelOneV1Enabled: boolean,
   isFlyout: boolean,
+  agentType?: ResponseActionAgentType,
```

Comment:
as `agentType` does not act only as a selected filter value, but results in a disabled filter that cannot be cleared, I'd suggest to indicate this in its name like `preselectedAgentType` or something else, in every file it's used